### ISSUE TITLE
Memory Leak

### ISSUE THREAD

**Issue by FroyoXSG** [oliverschwendener/ueli#605]
I only reboot my Windows machine when necessary, using Sleep mode otherwise.
There appears to be a memory leak in the latest version. I experienced it twice already, after not rebooting in a while.
As usual, memleak leads to excessive mem swapping and an unresponsive computer.

![ueli memory leak](https://user-images.githubusercontent.com/4962476/102480581-c28d2480-4060-11eb-8ffc-8b3e0c9a01f5.png)
![ueli memory leak2](https://user-images.githubusercontent.com/4962476/102480585-c325bb00-4060-11eb-9623-d447f0600501.png)

**Comment by oliverschwendener** [comment]
Probably related to #602

**Comment by tkohlmeier** [comment]
I am not sure this is really related because in #602 a powershell process
was affected, but here it is the ueli process itself. Could you verify this
by disabling the UWP plugin and check if the memory leak is gone then?

**Comment by FroyoXSG** [comment]
The UWP plugin was enabled for me. I'll try to keep the machine and ueli running. We'll see what happens.

**Comment by oliverschwendener** [comment]
@tkohlmeier I see, fair point.

**Comment by tkohlmeier** [comment]
@FroyoXSG There are multiple ueli processes. Next time you observe this issue, please go to details tab of task manager, enable the command line column and make a screenshot of all ueli processes with memory usage and command line:

![grafik](https://user-images.githubusercontent.com/6824291/102898665-1f764980-446a-11eb-8329-0376c9adbba7.png)

Then we know the process that is leaking memory.

**Comment by FroyoXSG** [comment]
It looks like I won't be able to provide you the screenshot, as the machine is currently unresponsive, but I can confirm that it's the render process leaking memory. It began while I was playing Shadow Warrior 2 in Fullscreen mode (DirectX 11 game).

**Comment by FroyoXSG** [comment]
![Capture](https://user-images.githubusercontent.com/4962476/103371886-ddd04900-4ad0-11eb-8f09-d0cbef752e0d.PNG)
Here's the screenshot now

**Comment by FroyoXSG** [comment]
I'm closing this issue because it's quite likely that a script that I used is to blame.
I updated ueli and other apps with a script that was broken in some cases (but not others).
If I don't follow up, assume that my broken script was the culprit.

Just in case anyone is interested, there's a [Superuser question](https://superuser.com/questions/1627572/my-updater-script-for-portable-apps-works-but-not-for-vscode) about my script woes.
